File: app/src/main/java/com/neome/api/meta/base/dto/EntVdExecuteParallel.java
// neome.ai API - do not change
//

package com.neome.api.meta.base.dto;

import com.neome.api.meta.base.Types.EnumDefnKindAutoResponseWait;

import org.jetbrains.annotations.Nullable;

@SuppressWarnings("unused")
public class EntVdExecuteParallel extends EntVdAutoStepWithError
{
  @Nullable
  public EnumDefnKindAutoResponseWait responseWaitKind;

  @Nullable
  public EntVdWorkflowControlMap workflowControlMap;
}

File: app/src/main/java/com/neome/api/meta/base/dto/EntVdExecuteParallel.kt
// neome.ai API - do not change
//
@file:Suppress("unused", "MemberVisibilityCanBePrivate", "CanBeParameter", "PropertyName")


package com.neome.api.meta.base.dto

import com.neome.api.meta.base.dto.EntVdAutoStepWithError
import com.neome.api.meta.base.dto.EntVdWorkflowControlMap
import com.neome.api.meta.base.Types.EnumDefnKindAutoResponseWait

interface EntVdExecuteParallel : EntVdAutoStepWithError
{
  val responseWaitKind: EnumDefnKindAutoResponseWait?
  val workflowControlMap: EntVdWorkflowControlMap?
}